Atmos QOL + Small Sprite Fix

13spacemen opened the pull request:
## About The Pull Request
Added screentips and balloon alerts to many atmos machines/pipes
Volume pump overclocking overlay is much slower and less seizure inducing
RPD screentips for right clicking pipes to copy settings
Removed (RPD) from the RPD's name since having an abbreviation in the name is ugly
Crystallizer and electrolyzer use ctrl+click and alt-click to turn on
On examine electrolyzer tells you about anchoring to drain from APC instead of cell
## Why It's Good For The Game
QOL for atmos stuff, user friendliness and better experience
## Changelog
:cl:
fix: Volume pump overclocking animation is much slower, no more headaches
qol: Added screentips to the RPD; screentips and balloon alerts to many atmos machines and devices
:cl:

13spacemen committed:
volume pump overclock flashing overlay is much slower
thermomachines have directional sprites

added screentip hints and balloon alerts to:
RPD, passive gate, pressure valve, pump, temperature gate, temperature pump, volume pump, electrolyzer, crystallizer, filter, mixer, bluespace gas sender, cryo cell, injector

Pickle-Coding commented:
![image](https://user-images.githubusercontent.com/58013024/227738785-0559f8bb-20ad-41a4-a058-3d48fd462b05.png)

Uhh, what does it look like in game?

Ghilker commented:
God those thermomachines' sprites are awful, make them properly.

13spacemen commented:
It looks fine in game, you're just not used to it. Looks normal and it's much easier to see what direction the freezer is
![freezers](https://user-images.githubusercontent.com/46101244/227739095-cb8f3028-51ca-4635-9ffa-4c1969718327.png)

Imaginos16 reviewed (changes requested):
This is some terrible directional perspective, I really dislike it.

Ghilker commented:
That doesn't look fine at all, just make the pipe that is sticking out more obvious instead of this monstrosity

Pickle-Coding commented:
> It looks fine in game, you're just not used to it. Looks normal and it's much easier to see what direction the freezer is
> 
> ![freezers](https://user-images.githubusercontent.com/46101244/227739095-cb8f3028-51ca-4635-9ffa-4c1969718327.png)

Thermomachines are in 3/4 perspective. You can't just rotate their sprite like that! Everything else in the PR is good though.

ZephyrTFA commented:
Upside down thermomachines are not real and cannot hurt you.

13spacemen committed:
reverted thermomachine sprites
added more balloon alerts, fixed some screentips

13spacemen commented:
Ok reverted the thermomachine sprites

LemonInTheDark commented on code/modules/atmospherics/machinery/components/unary_devices/thermomachine.dm:
```diff
@@ -43,6 +43,22 @@
+	. = ..()
+	context[SCREENTIP_CONTEXT_CTRL_LMB] = "Turn [on ? "off" : "on"]"
+	context[SCREENTIP_CONTEXT_ALT_LMB] = "Reset temperature"
+	if(held_item)
+		if(held_item.tool_behaviour == TOOL_SCREWDRIVER)
+			context[SCREENTIP_CONTEXT_LMB] = "[panel_open ? "Close" : "Open"] panel"
+		if(held_item.tool_behaviour == TOOL_WRENCH)
+			context[SCREENTIP_CONTEXT_LMB] = "Rotate"
+			context[SCREENTIP_CONTEXT_RMB] = "[anchored ? "Unan" : "An"]chor"
+		if(held_item.tool_behaviour == TOOL_MULTITOOL)
+			context[SCREENTIP_CONTEXT_LMB] = "Change piping layer"
+			context[SCREENTIP_CONTEXT_RMB] = "Change piping color"
+	return CONTEXTUAL_SCREENTIP_SET
```
early return, and switch on tool_behavior instead of an if chain

LemonInTheDark replied:
same applies in most uses of tool_behavior here

LemonInTheDark reviewed (changes requested):
3/4ths strikes again

13spacemen committed:
early returns and switches

LemonInTheDark commented on code/modules/atmospherics/machinery/components/gas_recipe_machines/crystallizer.dm:
```diff
@@ -39,6 +39,19 @@
+	. = ..()
+	if(!held_item)
+		return CONTEXTUAL_SCREENTIP_SET
+	context[SCREENTIP_CONTEXT_RMB] = "Turn [on ? "off" : "on"]"
```
shouldn't this be above the early return

13spacemen replied:
No because to turn the crystallizer off you need to attackby_secondary()
you can't turn it off with an empty hand

LemonInTheDark replied:
alllright, that's kinda silly, but alllright

13spacemen replied:
@LemonInTheDark you're right i just changed it rn, better to be consistent

13spacemen committed:
removed (RPD) from RPD name
electrolyzer and crystallizer use ctrl+click to turn on
more balloon alerts
renamed 'hatch' of electrolyzer to 'panel'

13spacemen committed:
changed electrolyzer on hotkey to Alt-click
